### PR TITLE
ADD search field and fix #51

### LINKED ISSUE #51: Update the Search Bar to work correctly
[Figma Link](https://www.figma.com/file/s0xuxeU6O2guoWEfA9OElZ/Bugheist-Full-Design?node-id=513%3A768)

search_bar

### PR DESCRIPTION
**Pull request by ankit2001**
This PR aims to add a search bar in the APP and fixing issue #51.

Please review it @souravbadami, @fredfalcon.
Thanks

### CODE REVIEW

**Review by fredfalcon — CHANGES_REQUESTED** [review]
This is not working right.  When I click search it brings me to the detail of the first issue.  This is supposed to search all issues and display a list of results.

**Comment by fredfalcon** [comment]
Also after it's clicked the bottom nav bar doesn't work anymore.

**Comment by ankit2001** [comment]
Will check it as well. Might be again some version error for crashing.

File: lib/pages/home.dart [hunk 1]
```diff
@@ -12,6 +14,33 @@ class Home extends StatefulWidget {
...
+  Icon _searchIcon = new Icon(Icons.search);
+  bool _ifSearching = false;
+  Widget _appBarTitle = new Image.asset(
+    'assets/bugheist_logo.png',
+    fit: BoxFit.cover,
+    height: 30,
+  );
+  void _getNames() async {
+    final response =
+        await get(Uri.parse('https://www.bugheist.com/api/v1/issues/'));
```
Comment (fredfalcon) [inline lib/pages/home.dart:30]: please use the url from the constants file

File: lib/pages/home.dart [hunk 2]
```diff
@@ -12,6 +14,33 @@ class Home extends StatefulWidget {
...
+    final Map<String, dynamic> searchData = json.decode(response.body);
+    List<dynamic> tempList = [];
+    for (int i = 0; i < searchData['results'].length; i++) {
+      tempList.add(searchData['results'][i]);
+      //print(tempList[i]);
```
Comment (fredfalcon) [inline lib/pages/home.dart:35]: remove any comments

**Review by fredfalcon — CHANGES_REQUESTED** [review]

**Comment by fredfalcon** [comment]
@ankit2001 can you please update this to work ?

**Comment by ankit2001** [comment]
> 

Sure, will fix all changes, this week.

**Pull request closed without merging**